package/yast2-dns-server.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Apr 27 09:33:03 UTC 2015 - ancor@suse.com
+
+- Experiments
+- 3.1.11.1
```

Comment:
do not make fake entries and do not create wrong versions, it is just confusing ;)